lib/errors.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const getContent = (err, debug, translate) => {
+  var content = {};
+
+  if (err.code === 'SESSION_TIMEOUT') {
+    content.title = translate('errors.session.title');
```

Comment:
I can't see much of an alternative here. Unless we want to have the translations provided directly instead of the translation library?

At least we aren't prescribing which translation lib they use, and they can opt not to make use of the middleware is they choose.